File: inspect_messages.py
import json, os
path = os.path.join('backend','data','coordinatorGraphStore.json')
print('exists', os.path.exists(path))
print('path', path)
with open(path, 'r', encoding='utf8') as f:
    data = json.load(f)
msgs = data.get('messages', [])
print('messages count', len(msgs))
print(json.dumps(msgs[:10], indent=2))

File: inspect_messages.mjs
import { getDB } from './backend/mongoClient.js'

const db = await getDB()
const msgs = await db.collection('messages').find({}).sort({ created_at: -1 }).toArray()
console.log('messages count', msgs.length)
console.log(JSON.stringify(msgs.slice(0, 10), null, 2))
